cra-tf-validate-ignore-rules.json

```diff
@@ -8,7 +8,7 @@
     },
```

Comment:
can't we now remove `rule-3b2768e5-d783-4b0c-a47f-81479af34689` ?

Reply:
Ok, so please:

- remove `Found in: resource_address: module.event_streams.ibm_resource_instance.es_instance"` from the description
- update the `ignore_reason` to: Failing due to known CRA bug (https://github.ibm.com/oneibmcloud/CD-CRA/issues/1820)
- `"is_valid": false` since this is not a valid compliance failure - we are setting a private only endpoint

Reply:
Done